### PR TITLE
Stage1

### PR DESCRIPTION
**Pull request by sayyamgatagat**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1>Check My pic</h1>
     <footer>
         <p>Sayyam Gatagat. © 2020</p>
     </footer>
-    <script src="JS/count.js">
+    <script src="/checkpic.js">
     </script>
```
Comment (gvsvarun) [inline index.html:23]: no need of end tag

Comment (gvsvarun) [reply index.html:23]: you can also resolve the changes to master

Comment (sayyamgatagat) [reply index.html:23]: ok will update it

Comment (gvsvarun) [reply index.html]: For some reason I still see conflicts. Can you fix that.

Comment (sayyamgatagat) [reply index.html]: what conflict?

**Pull request closed without merging**